ForgeUnity/Assets/BeardedManStudios/Scripts/Multiplayer Menu/MultiplayerMenu.cs

```diff
@@ -219,6 +223,20 @@ private void Update()
 				NetWorker.localServerLocated += TestLocalServerFind;
 				NetWorker.RefreshLocalUdpListings();
 			}
+#endif
+
+#if ENABLE_INPUT_SYSTEM //if using new Input System
+			if (Keyboard.current.hKey.isPressed)
+				Host();
+			else if (Keyboard.current.cKey.isPressed)
+				Connect();
+			else if (Keyboard.current.lKey.isPressed)
+            {
+				NetWorker.localServerLocated -= TestLocalServerFind;
+				NetWorker.localServerLocated += TestLocalServerFind;
+				NetWorker.RefreshLocalUdpListings();
+			}
+#endif
```

Comment:
This should leave the legacy Input code intact, so users who haven't migrated are unaffected.